Earthfile

```diff
@@ -338,6 +346,27 @@ base-image:
 
     # END
 
+    # Build with provider-kairos
+    # TODO: Allow installing from luet packages
+    ARG PROVIDER_KAIROS
+    ARG PROVIDER_KAIROS_BRANCH=main
+    IF [[ "$PROVIDER_KAIROS" != "" ]]
+        DO github.com/kairos-io/provider-kairos:main+PROVIDER_INSTALL --FLAVOR=$FLAVOR
+        # Redo os-release with override settings to point to provider-kairos stuff
+        # in earthly 0.7 we will be able to just override VARIANT here and just run the OSRELEASE once
```

Comment:
could we just upgrade to this verstion already?

Reply:
Not easily as 0.7 introduces the concept of global vars and that could potentially screw the current earthfile. 

It would need to come in a different PR to make sure we don't break any other stuff